index.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+/**
+ * textarea-feedback Input Plugin
+ *
+ *
+ * A simple feedback textarea input type
+ */
+const validation = require('./lib/validation.js');
+const counter = require('./lib/charCounter.js');
+
+/**
+ * Single textarea-feedback Input Plugin
+ * @module textareaFeedbackInputPlugin
+ */
+module.exports = {
+  name: 'textarea-feedback',
+  description: 'A simple feedback textarea input type',
+  validation: {
+    textareaFeedbackValidation: validation,
+  },
+  scripts: {
+    characterCounter: counter,
+  },
+  inputs: {
+    textareaFeedback: {
+      scripts: {
+        function: 'characterCounter',
+
+      },
+      validation: {
+        function: 'textareaFeedbackValidation',
+        on: 'blur',
+      },
+      label: 'Please enter some text',
+      type: 'textarea',
+      value: 'type here...',
```

Comment:
Do not include `value`. That is the actual value that will be saved/entered by the user. If this is placeholder text, have it be `placeholder`